tests/components/blebox/test_cover.py

```diff
@@ -391,10 +391,10 @@ async def test_entity_properties(feature, hass, config):
     entity = BleBoxCoverEntity(feature_mock)
 
     feature_mock.state = 1
-    assert entity.is_opening is True
+    assert entity.state == STATE_OPENING
```

Comment:
Get the state from the core state machine after setting up the integration.

Reply:
Yes, of course. Done.